android/app/src/main/kotlin/com/jabook/app/jabook/audio/PlaylistInitialPositionPolicy.kt

```
@@ -0,0 +1,40 @@
+internal data class PlaylistInitialPositionDecision(
+    val shouldScheduleDeferredApply: Boolean,
+    val normalizedTargetTrackIndex: Int?,
+)
+
+internal object PlaylistInitialPositionPolicy {
+    internal fun decidePostPrepare(
+        requestedTrackIndex: Int?,
+        requestedPositionMs: Long?,
+        playlistSize: Int,
+    ): PlaylistInitialPositionDecision {
+        if (requestedTrackIndex == null || requestedPositionMs == null || requestedPositionMs <= 0L || playlistSize <= 0) {
```

detekt (1.23.8)

[warning] This condition is too complex (4). Defined complexity threshold for conditions is set to '4'

(detekt.complexity.ComplexCondition)